website/docs/reference/project-collaboration-mode.md

```diff
@@ -2,52 +2,64 @@
 title: Project Collaboration Mode
 ---
 
-:::info Availability
+:::note Availability
 
-The project collaboration mode is an enterprise-only feature that was introduced in **Unleash 4.22.0**.
+**Plan**: [Enterprise](https://www.getunleash.io/pricing) | **Version**: `4.22+`
 
 :::
 
-A project's **collaboration mode** specifies **who can submit [change requests](change-requests.md)**. There are two collaboration modes:
-- [**open**](#open-collaboration-mode)
-- [**protected**](#protected-collaboration-mode)
-- [**private**](#private-collaboration-mode)
+# Overview
 
-## **Open** collaboration mode
+Project collaboration modes allow you to manage project visibility and interaction levels, ensuring better control and collaboration within your organization.
 
-Anyone can submit change requests in an **open** project, regardless of their permissions within the project and globally. This is the default collaboration mode for projects.
+The collaboration mode of a [project](projects.md) defines who within your Unleash instance can access the project. This setting also determines who can submit [change requests](change-requests.md)–a feature that provides controlled and secure updates for feature toggles.
 
-The open collaboration mode is the default in Unleash and is how all projects worked in Unleash before the introduction of collaboration modes.
+Unleash supports three collaboration modes: **open**, **protected**, and **private**. The [predefined root and project roles](rbac.md#predefined-roles), as well as [custom root roles](rbac.md#custom-root-roles) of a user, determine what projects they can access.
```

Comment:
Maybe consider restructuring the sentence here, so that "user" shows up earlier? I think it'd make it easier to understand what roles we're talking about (at least I was a little confused when reading this at first).

Maybe something like this? 

```suggestion
Unleash supports three collaboration modes: **open**, **protected**, and **private**. A user's [predefined root and project roles](rbac.md#predefined-roles), as well as their [custom root roles](rbac.md#custom-root-roles), determine what projects they can access.
```